### DISCUSSION TITLE
Mirth 4.3.0 and CVE-2023-37679?

### DISCUSSION THREAD

**Discussion by bfnoling** [nextgenhealthcare/connect#5892] [General]
While doing research on an unrelated vulnerability, I came across this:

https://nvd.nist.gov/vuln/detail/CVE-2023-37679

I get a cloudflare error when trying to click the third party link and don't see any other information outside of the NIST and a Github Advisory Database entry that is identical to what is on NIST.

I checked the release notes and completed milestones for 4.4.0 and do not see mention of any security fixes.

Anyone have any idea what this about?

**Comment by narupley** [comment]
Yes, the person who created that CVE disclosed it to us beforehand, and we already fixed it in 4.4! It is here in the release notes: https://github.com/nextgenhealthcare/connect/wiki/4.4.0---What's-New#remote-code-execution-vulnerability-when-using-java-8-and-elevated-permissions

Quoting the release notes here:
> This doesn't affect any server running Java 9+, it only happens on Java 8, and only when the Mirth Connect process is given sufficient elevated OS permissions. Java 11 at a minimum is recommended, and Java 8 support will be phased out at some point. But if you do absolutely need to use Java 8, then we recommend upgrading to Mirth Connect 4.4 or later.

As a reminder, we have an e-mail that anyone can use to privately disclose vulnerability reports to us! That is what happened in this case, so I'm glad we set that up. It's here in our security policy: https://github.com/nextgenhealthcare/connect/security/policy
